src/python/pypresso.cmakein

```diff
@@ -17,13 +17,19 @@ export PYTHONPATH
 
 if [ "$1" = "--gdb" ]; then
   shift
-  exec gdb -ex run --args @PYTHON_FRONTEND@ $@
+  exec gdb --args @PYTHON_FRONTEND@ $@
 elif [ "$1" = "--lldb" ]; then
   shift
-  exec lldb -o "run $@" @PYTHON_FRONTEND@
+  exec lldb -- @PYTHON_FRONTEND@ $@
+elif [ "$1" = "--valgrind" ]; then
+  shift
+  exec valgrind --leak-check=full @PYTHON_FRONTEND@ $@
```

Comment:
Need to be able to optionally add extra arguments to `valgrind` as it is not interactive like the debuggers.

Reply:
To be honest, if someone needs to add extra arguments to `valgrind`, this person is probably smart enough to open the script and extract the relevant stuff.  And I think `--leak-check=full` is most common.

Reply:
Smart people can be lazy too. pypresso is all about convenience. That's why I'd like to be able to do something like this: `./pypresso --valgrind='--leak-check=full --verbose --trace-children=yes --time-stamp=yes' mysim.py 1.0 1.0 3.1 0.7`. If no arguments are specified, your default of `--leak-check=full` should be used.

Reply:
The syntax you requested should be feasible now.  I'm not a Bash wizard so my solution comes with quite a bit of code duplication.